Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/BoundsControl.cs

```diff
@@ -862,8 +856,6 @@ private void DetermineTargetBounds()
             }
 
             TargetBounds.size += Vector3.Scale(boxPadding, scale);
-
-            TargetBounds.EnsureComponent<NearInteractionGrabbable>();
```

Comment:
Will this be a breaking change? If I don't currently have a `NearInteractionGrabbable` on my GameObject and rely on this adding one at runtime?

Reply:
Ensure component adds the component if it doesn't exist, otherwise it uses the existing one on the gameobject

Reply:
But in this scenario, there _isn't_ an existing one on the GameObject. Before this PR, one was created for me at runtime, but after this PR, it won't, right?

Reply:
No it will, EnsureComponent adds it at run time if it doesn't exist when that line is hit.

Reply:
https://github.com/microsoft/MixedRealityToolkit-Unity/blob/265a0ea3621c84b88e4b02e7fcd1b5682afef56a/Assets/MRTK/Core/Extensions/ComponentExtensions.cs#L61-L65

Reply:
But this PR _removes_ the `EnsureComponent` line. Or is it just being moved and I missed it?

Reply:
Ohhhhh I missed the original context (I thought you were referring to the bounding box).

Yeah this'll be breaking for anyone who relies on this script to add the NearInteractionGrabbable, but I think that was a mistake on the part of the Bounding Box in the first place, since there's nothing about this component that then necessitates that near interaction be available. It's in a similar position as ObjectManipulator in my opinion, which doesn't add a NearInteractionGrabbable.

Reply:
Ultimately deciding on removing `EnsureComponent<NearInteractionGrabbable>` from BoundsControl. The reason being that the individual handles already have NearInteractionGrabbable components, enabling NearInteraction with the BoundsControl regardless of whether the root TargetBounds object has BoundsControl. 

Nothing in BoundsControl explicitly depends on a BoundsControl component on the TargetBounds gameobject, so removing this line will make it so we don't add vestigial components without developer intent.